src/python/grpcio_tests/tests/unit/_server_ssl_cert_config_test.py

```diff
@@ -94,18 +68,45 @@ def __call__(self):
             return ret
 
 
-class _ServerSSLCertReloadTest(object):
+class _ServerSSLCertReloadTest(six.with_metaclass(abc.ABCMeta, unittest.TestCase)):
+
+    def __init__(self, *args, **kwargs):
+        super(_ServerSSLCertReloadTest, self).__init__(*args, **kwargs)
+        self.client_key_1_pem = resources.cert_hier_1_client_1_key()
```

Comment:
What about these constants (they are constants, aren't they?) earns them promotion from module-scope to instance-scope? What was wrong with their being at module-scope?

Reply:
sure; i don't expect them to change during the tests, so i can make them module-scope, with ALL_UPPERCASE_NAMES

Reply:
done